src/test/java/com/puppycrawl/tools/checkstyle/checks/indentation/IndentationCheckTest.java

```diff
@@ -668,8 +668,8 @@ public void testInvalidMethodWithChecker()
             "110:15: " + getCheckMessage(MSG_ERROR, "operator new", 14, 16),
             "113:11: " + getCheckMessage(MSG_CHILD_ERROR, "method call", 10, 12),
             "118:15: " + getCheckMessage(MSG_ERROR, "operator new", 14, 16),
-            "122:11: " + getCheckMessage(MSG_ERROR, "new", 10, 12),
-            "126:11: " + getCheckMessage(MSG_ERROR, "new", 10, 12),
+            "122:11: " + getCheckMessage(MSG_ERROR, "operator new", 10, 12),
```

Comment:
diff report is heavily polluted by such update

can we avoid such wording update ?

Reply:
I guess no.
Since Inside the newHandler the message is `new operator` and inside linewrapping it was only `new`.
Though its our choice to choose between `new` or `new operator`, but the diff report will still be polluted for once atleast.

Reply:
https://github.com/checkstyle/checkstyle/blob/432bafd49ed9d801f44a04ad710cc9372538e588/src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/NewHandler.java#L42-L47

> inside linewrapping it was only new

can you share exact line where it is defined ?

Reply:
please help to remove this word from message in scope of https://github.com/checkstyle/checkstyle/issues/9113
it will simplify regression report review.
You can place update for message in this PR as separate commit so it will be easy to regenerate report.

Unfortunately Indentation Check is so complicated and so many variations of code could be so we rely of diff report too much, as  our test have only very basic set of inputs.
Current diff report is unmanageable, too much changes.

Reply:
please help to remove this word from message in scope of https://github.com/checkstyle/checkstyle/issues/9113
it will simplify regression report review.
You can place update for message in this PR as separate commit so it will be easy to regenerate report.

Unfortunately Indentation Check is so complicated and so many variations of code could be so we rely of diff report too much, as  our test have only very basic set of inputs.
Current diff report is unmanageable, too much changes.

Reply:
Doing it, `operator new` to `new`
Generating the report at this point.

Reply:
Done